project/1.18/assets/tinkers-construct/tconstruct/lang/zh_cn.json

```
@@ -1294,7 +1331,7 @@
   "material.tconstruct.wood.crimson": "绯红菌核",
   "material.tconstruct.wood.warped": "诡异菌核",
   "material.tconstruct.wood.bamboo": "竹子",
-  "material.tconstruct.wood.format": "木%s",
+  "material.tconstruct.wood.format": "木 %s",
```

这里去掉空格是因为原文用的是“Wooden”后接材料名称，而不是其他材料那样直接沿用了材料名（木的材料名是Wood），导致在游戏内所有木制部件都会平白无故多一个空格。
```suggestion
  "material.tconstruct.wood.format": "木%s",
```

---

这里有点出入，不过我印象比较深刻的是磨制工具，别的材料没有空格就木头有